pkg/ccl/changefeedccl/testfeed_test.go

```diff
@@ -906,6 +925,32 @@ func (f *cloudFeedFactory) Feed(
 		)
 	}
 
+	parquetPossible := true
+
+	for _, opt := range createStmt.Options {
+		if string(opt.Key) == changefeedbase.OptFormat {
+			parquetPossible = false
+			break
+		}
+		for o := range changefeedbase.InitialScanOnlyUnsupportedOptions {
+			if o == string(opt.Key) {
+				parquetPossible = false
+				break
+			}
+		}
+	}
+
+	if parquetPossible {
+		TestingSetIncludeParquetMetadata()
```

Comment:
you need to invoke returned function -- via defer.
Unfortunately, in this case, it would immediately reset this setting.
You probably need to have this cleanup function invoked when feed shuts down.

You could, of course, call this function unconditionally in https://github.com/cockroachdb/cockroach/blob/master/pkg/ccl/changefeedccl/helpers_test.go#L823-L823 (c makeFeedFactoryWithOptions) -- since that function already returns cleanup, you can simply invoke function to undo this setting.